src/main/java/no/nav/pto/veilarbportefolje/oppfolgingsbruker/OppfolgingsbrukerRepositoryV2.java

```diff
@@ -123,4 +129,35 @@ private OppfolgingsbrukerEntity mapTilOppfolgingsbruker(ResultSet rs, int row) {
                 rs.getBoolean(HAR_OPPFOLGINGSSAK), rs.getBoolean(SPERRET_ANSATT), rs.getBoolean(ER_DOED),
                 toZonedDateTime(rs.getTimestamp(DOED_FRA_DATO)), toZonedDateTime(rs.getTimestamp(ENDRET_DATO)));
     }
+
+    public List<String> hentBrukereSomSkalSensureres(List<String> fnrListe, boolean tilgangTilKode6, boolean tilgangTilKode7, boolean tilgangTilEgenAnsatt) {
+        String skjermetDiskresjonskoder = hentDiskresjonskoderSomSkalSensureres(tilgangTilKode6, tilgangTilKode7);
```

Comment:
I don't like this inverse logic, its hard to read. Is it possible to fetch all records that dont have  diskresjonskode and records that have diskresjonskode which is allowed to read by veileder.

Reply:
I should be better now 👍